languages/scala/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,105 @@
+## Defining Variables
+
+Defining a variable means assigning a value to a name. In Scala there are two kinds of variables, vals and vars.
+
+A `val` is [immutable][immutability], so it's value cannot be changed anymore after the first assignment:
```

Comment:
In general, we try to not link to other resources from the `introduction.md` document. While this is a document that we wrote and thus is relatively basic, there is quite a bit of information in the linked document that the student has no need for (and will probably confuse the student), such as the implementing exercises and such.

```suggestion
A `val` is immutable, so it's value cannot be changed anymore after the first assignment:
```

Note that in the `hints.md` and `after.md` documents, we _do_ encourage linking to external resources.

Reply:
Makes sense, I removed them.